events/courtier_guest_management_events/courtier_guest_management_events.txt

```diff
@@ -828,6 +828,10 @@ scripted_trigger courtier_guest_management_3001_interesting_trigger = {
 		has_trait = intellect_good
```

Comment:
Looks like this needs adaptation.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105053045-f86f5f80-5a89-11eb-82bc-9a4a4c021127.png)

</details>
I would add scripted triggers here.